vmdb/config/vmdb.tmpl.yml

```diff
@@ -284,6 +284,7 @@ workers:
           :cinder: "notifications.*"
           :glance: "notifications.*"
           :quantum: "notifications.*"
+          :ironic: "conductor_manager.*"
```

Comment:
@Fryguy Also, to your point, yes, this should really go into the _infra nested catcher, not at this level -- I'll keep that in mind when I add it back into my dev env.